test/testdata

```diff
@@ -199,37 +199,38 @@ Scan      ORCA         OrcaScanTest                   basicORCA3.0          dvb_
-SP        Psi         PsiSPTest             basicPsi4.0           dvb_sp_rks.out
-SP        QChem       GenericSPTest         basicQChem4.2         dvb_sp.out
+SP        ADF         ADFSPTest             basicADF2007.01             dvb_sp_b.adfout
+SP        ADF         ADFSPTest             basicADF2013.01             dvb_sp_b.adfout
+SP        DALTON      GenericSPTest         basicDALTON-2013            dvb_sp_hf.out
+SP        DALTON      GenericSPTest         basicDALTON-2013            dvb_sp_ks.out
+SP        DALTON      GenericSPTest         basicDALTON-2015            dvb_sp_hf.out
+SP        DALTON      GenericSPTest         basicDALTON-2015            dvb_sp_ks.out
+SP        GAMESSUK    GenericSPTest         basicGAMESS-UK7.0           dvb_sp_b.out
+SP        GAMESSUK    GenericSPTest         basicGAMESS-UK8.0           dvb_sp_hf.out
+SP        GAMESSUK    GenericSPTest         basicGAMESS-UK8.0           dvb_sp_ks.out
+SP        GAMESS      GenericSPTest         basicFirefly8.0             dvb_sp.out
+SP        GAMESS      GenericSPTest         basicGAMESS-US2014          dvb_sp.out
+SP        GAMESS      GenericSPTest         basicGAMESS-US2017          dvb_sp.out
+SP        Gaussian    GaussianSPTest        basicGaussian03             dvb_sp.out
+SP        Gaussian    GaussianSPTest        basicGaussian09             dvb_sp.out
+SP        Gaussian    GaussianSPTest        basicGaussian16             dvb_sp.out
+SP        Jaguar      Jaguar7SPTest         basicJaguar7.0              dvb_sp.out
+SP        Jaguar      JaguarSPTest          basicJaguar8.3              dvb_sp_hf.out
+SP        Jaguar      JaguarSPTest          basicJaguar8.3              dvb_sp_ks.out
+SP        Molcas      MolcasSPTest          basicOpenMolcas18.0         dvb_sp.out
+SP        Molpro      MolproSPTest          basicMolpro2006             dvb_sphf.out
+SP        Molpro      MolproSPTest          basicMolpro2012             dvb_sphf.out
+SP        NWChem      GenericSPTest         basicNWChem6.0              dvb_sp_hf.out
+SP        NWChem      NWChemKSSPTest        basicNWChem6.0              dvb_sp_ks.out
+SP        NWChem      GenericSPTest         basicNWChem6.5              dvb_sp_hf.out
+SP        NWChem      NWChemKSSPTest        basicNWChem6.5              dvb_sp_ks.out
+SP        ORCA        OrcaSPTest            basicORCA2.9                dvb_sp.out
+SP        ORCA        OrcaSPTest            basicORCA3.0                dvb_sp.out
+SP        Psi         Psi3SPTest            basicPsi3                   dvb_sp_hf.out
+SP        Psi         PsiSPTest             basicPsi4.0                 dvb_sp_rhf.out
+SP        Psi         PsiSPTest             basicPsi4.0                 dvb_sp_rks.out
+SP        QChem       GenericSPTest         basicQChem4.2               dvb_sp.out
+SP        Turbomole   GenericSPTest         basicTurbomole5.9/dvb_sp    basis         control      job.last      coord      mos
```

Comment:
What do we think is better, this or
```
SP Turbomole GenericSPTest basicTurbomole5.9 dvb_sp/basis dvb_sp/control dvb_sp/job.last dvb_sp/coord dvb_sp/mos
```
?

Reply:
I think dvb/filenames make it more intuitive. Will change it accordingly.

Reply:
I don't have an opinion here.